CHANGELOG.md

```diff
@@ -57,6 +57,7 @@
 - Cleanup metric names inside scalers ([#2260](https://github.com/kedacore/keda/pull/2260))
 - Validating values length in prometheus query response ([#2264](https://github.com/kedacore/keda/pull/2264))
 - Add `unsafeSsl` parameter in SeleniumGrid scaler ([#2157](https://github.com/kedacore/keda/pull/2157))
+- GCP PubSub scaler may be used in SubscriptionSize and OldestUnackedMessageAge modes
 
 ### Breaking Changes
 
```

Comment:
Might be good to add the deprecation in our changelog
```suggestion
- GCP PubSub scaler may be used in SubscriptionSize and OldestUnackedMessageAge modes

### Deprecations

- `subscriptionSize` is deprecated in favor of `mode` and `value` for GCP Pub/Sub scaler

### Breaking Changes

```

@zroubalik We should open a notice when we ship vNext

Reply:
good point, committed the suggestion